packages/augur-core/src/contracts/trading/erc20proxy1155/IERC20Proxy1155Nexus.sol

```diff
@@ -0,0 +1,40 @@
+pragma solidity 0.5.15;
+
+import 'ROOT/libraries/CloneFactory.sol';
+import 'ROOT/libraries/token/IERC20.sol';
+import 'ROOT/libraries/token/IERC1155.sol';
+import 'ROOT/libraries/math/SafeMathUint256.sol';
+
+import 'ROOT/trading/erc20proxy1155/IERC20Proxy1155.sol';
+
+// To use these ERC20s, the user will have to call
+// setApprovalForAll(ERC20Proxy1155Nexus, true)
+// on the ShareToken contract themselves.
+
+contract IERC20Proxy1155Nexus is CloneFactory {
```

Comment:
The actual ERC20Proxy1155Nexus should inherit from CloneFactory instead of this interface. That'll keep imports of the interface trim and implementation out of an interface

Reply:
oh right!